src/extensions/editable/bootstrap-table-editable.js

```diff
@@ -48,11 +48,10 @@ $.BootstrapTable = class extends $.BootstrapTable {
       }
 
       const editableOptions = {}
-      const editableDataMarkup = []
       const editableDataPrefix = 'editable-'
       const processDataOptions = (key, value) => {
         // Replace camel case with dashes.
-        const dashKey = key.replace(/([A-Z])/g, $1 => `-${$1.toLowerCase()}`)
+        const dashKey = key.replace(/([A-Z])/g, $1 => "-" + $1.toLowerCase())
```

Comment:
I don't think we need that change, as we optimize/transpile the source code on a new release (see dist folder).
Also we use templated strings everywhere in the project, so if it makes sense to refactor it we should do it in the whole project.

Reply:
reverted to original in this line. (Don't remember why I downgraded syntax to ES5, this was half year ago.)

Reply:
Oh, I remembered! This because I modified file in "dist" folder which has ES5 syntax! Anyway this line reverted now.